backend/src/main/java/hu/flowacademy/worksheet/controller/UserController.java

```diff
@@ -44,9 +47,21 @@ public AccessTokenResponse login(@RequestBody UserOperationDTO userOperationDTO)
         return keycloakClientService.login(userOperationDTO.getEmail(), userOperationDTO.getPassword());
     }
 
+    private final PagingProperties pagingProperties;
+
+    @RolesAllowed("admin")
     @GetMapping("/users")
-    @ResponseStatus(HttpStatus.FOUND)
-    public List<User> findUserByNameOrEmail(@RequestParam(value = "q") String q) {
-        return userService.findUserByNameAndEmail(q);
+    public List<User> getRegistrations(@RequestParam(value = "page", required = false) Optional<Integer> page,
+                                       @RequestParam(value = "limit", required = false) Optional<Integer> limit,
+                                       @RequestParam(value = "order_by", required = false) Optional<String> orderBy) {
+        if (page.isPresent()) {
```

Comment:
Ezek a feltétel vizsgálatok kerüljenek a service-be, ott dőljön el, hogy attól függően, hogy vannak-e lapozást segítő paraméterek, lapozzunk-e, illetve rendezzünk-e. A keresés paraméterét (q) is küld át és ha van értéke, akkor szűrjünk arra is, ezen úgy tudom @nomansnick dolgozik rajta.

Reply:
Patrické májd teljesen más lesz, még akkor nem tudom mit rakjak hova. Mert változni fog

Reply:
Az enyémet viszont átdobtam a service-be és használható